resources/sdk/purecloudgo/templates/configuration.mustache

```diff
@@ -506,16 +514,17 @@ func (c *Configuration) AuthorizeClientCredentials(clientID string, clientSecret
 func (c *Configuration) AuthorizeCodeGrant(clientID string, clientSecret string, authCode string, redirectUri string) (*AuthResponse, error) {
 	c.ClientID = clientID
 	c.ClientSecret = clientSecret
-	authHostRegex := regexp.MustCompile(`(?i)\/\/api\.`)
-	authHost := authHostRegex.ReplaceAllString(c.BasePath, "//login.")
```

Comment:
here the way we are constructing the url , is different from gc cli. better we dont change this logic and use the same in getconfurl
for the basepath